build/docker-entrypoint.sh

```diff
@@ -42,8 +42,8 @@ if [ "$#" -eq 0 ]; then
   if [ -x "$(which ansible)" ]; then
     echo -e "ansible\t\t ${ANSIBLE_VERSION}"
   fi
-  current_version=$(cat /home/easy_infra/.tfenv/version)
   if [ -x "$(which terraform)" ]; then
+    current_version=$(cat /home/easy_infra/.tfenv/version)
```

Comment:
This fixes a bug we never noticed before because `terraform` in all images, the `cat` would fail if `tfenv` wasn't installed and setup because `tfenv`/`terraform` isn't installed.